doc/ReleaseProcess.md

```diff
@@ -0,0 +1,168 @@
+2. Make sure the venv is activated: `. venv/bin/activate`
+3. Install packages from `requirements.txt`: `pip install -r requirements.txt`
+4. From project root, run the unit tests:
+
+    python -m unittest simplegallery/test/*.py
+
+This will generate a lot of output. Usually it is obvious if a test fails. Scan
+the console output to make sure all the test ran okay.
+
+### Integration Test
+
+The integration test will use a clean venv and install the distribution package
+you built earlier, and run some tests against it.
+
+1. Deactivate the previous venv: `deactivate`
+2. Switch to the test gallery directory: `cd examples/test_gallery`
+3. Set the version of the package you are testing: `export SPG_VER=1.2.3`
+4. `make help` to verify that the version is correct (the help text will show
+   the value for `venvdist`)
+5. Create the test venv, but using the distribution package instead of the
+   default: `make venvdist`
+
+Now you have created a venv using the SPG distribution package. This is the
+same package that will be published on PyPI, assuming all tests are okay. Run
+tests as described in the [Test Gallery README](../examples/test_gallery/README.md).
+
+Here is the short version that does not include any remote galleries or
+deployment tests. This is an example to show the process of setting up and
+running some tests, but probably you should do more than this for a release:
+
+```
+deactivate              # make sure no other venv is active
+cd examples/test_gallery
+export SPG_VER=1.2.3    # use the actual version, obviously
+make clean              # clean out old tests
+make cleanvenv
+make venvdist
+make testinit
+make testbuild
+```
+
+There should be no error messages generated. Examine the static gallery under
+`output/public`.
+
+You can also open `output/public/index.html` with a browser to verify the
+generated site. Or instead `make serve` and open a browser to
+`http://localhost:8008`.
+
+## Publish
+
+At this stage everything above should be done:
+
+* the version in `setup.py` was updated
+* built packages with the new version in the `dist` directory
+* committed (and pushed) the updated `setup.py`
+* HEAD commit is tagged with the version (and the tag is pushed)
+* testing was done and you are satisfied with test result
+
+### Github Release
+
+**Note:** before you start, it is a good idea to create a temporary text file
+with the release notes. Look at the commit history to see all the changes that
+were made, and make a list of bugs (issue numbers) that were fixed. This will
+make it easier when you get to the step below for filling out the release
+description.
+
+Create a release on github from the new version tag you pushed (assumes you
+are a maintainer of this project on github):
+
+1. https://github.com/haltakov/simple-photo-gallery
+2. go to the releases page
+3. click "draft a new release"
+4. under the "choose a tag" dropdown, select the new release tag
```

Comment:
Small hint - at this stage GitHub allows you to create a new tag as well. You can save one step from before where you create the tag with git

Reply:
I made changes per your suggestion. See what you think.